Sources/CryptoSwift/Salsa20.swift

```diff
@@ -0,0 +1,262 @@
+//
+//  Salsa20.swift
+//  CryptoSwift
+//
+//  Created by Dennis Michaelis on 27.03.16.
+//  Copyright © 2016 Dennis Michaelis. All rights reserved.
+//
+
+public class Salsa20 {
+    
+    public enum Error: ErrorType {
+        case MissingContext
+        case LimitOfNonceExceeded
+        case NotImplementedYet
+    }
+    
+    static let blockSize = 64 // 512 / 8
+    internal let stateSize = 16
```

Comment:
since internal is default, better leave it this way (aka. `internal` is redundant here)